core/PRU.cpp

```diff
@@ -1514,15 +1518,15 @@ void PRU::waitForFinish()
 void PRU::disable()
 {
     /* Disable PRU and close memory mapping*/
-    prussdrv_pru_disable(pru_number);
+    pruManager->stop();
```

Comment:
while you are at it, please replace the leading spaces in this and the above line with a tab.